nautobot/ipam/models.py

```diff
@@ -645,7 +638,11 @@ def get_available_ips(self):
         available_ips = prefix - child_ips
 
         # IPv6, pool, or IPv4 /31-32 sets are fully usable
-        if self.family == 6 or self.is_pool or (self.family == 4 and self.prefix.prefixlen >= 31):
+        if (
+            self.family == 6
+            or self.type == choices.PrefixTypeChoices.TYPE_POOL
+            or (self.family == 4 and self.prefix.prefixlen >= 31)
+        ):
```

Comment:
IMO when it comes to multiple `or` matches, I find that using `any()` is more readable:
```suggestion
        if any([
            self.family == 6,
            self.type == choices.PrefixTypeChoices.TYPE_POOL,
            self.family == 4 and self.prefix.prefixlen >= 31,
        ]):
```